### PR TITLE
Update Foundation's Delegation Policy

### PR DESCRIPTION
**Pull request by tjanez**

### CODE REVIEW

**Review by kostko — APPROVED** [review]

File: docs/general/foundation/delegation-policy.md [hunk 1]
```diff
@@ -4,37 +4,60 @@ description: This page describe the Oasis Protocol Foundation's Delegation Polic
...
-* Engagement in network governance.
-* Actively engaging in the community slack workspace and helping answer questions from other community members.
-* Building tools, DApps or services that benefit the developers, node operators and/or the overall community.
-* Finding and reporting issues related to any of the network protocols and/or their implementations.
+Oasis Protocol Foundation delegates ROSE tokens to node operators based on their
+participation in the Oasis Mainnet, Testnet and ParaTimes on the network,
+node reliability and performance, community engagement and overall support of
+the network.
+
+## Requirements for receiving delegations
```
Comment (peterjgilbert) [inline docs/general/foundation/delegation-policy.md]:
```suggestion
## Requirements for Receiving Delegations
```

File: docs/general/foundation/delegation-policy.md [hunk 2]
```diff
@@ -4,37 +4,60 @@ description: This page describe the Oasis Protocol Foundation's Delegation Polic
...
+as possible but no later than the 3 business day window.
+
+## How to earn delegations?
+
+- Have a meaningful amount of self-delegation and/or delegations from the Oasis
+  community.
+- Operate Emerald and/or Cipher ParaTime nodes on Mainnet.
+- Operate validator and/or ParaTime nodes on Testnet
+- Participate in community discussion on the network’s future roadmap.
+- Actively participate in the Oasis Community Slack and Oasis Discord channel
```
Comment (peterjgilbert) [inline docs/general/foundation/delegation-policy.md]:
```suggestion
- Actively participate in the Oasis Community Slack and Oasis Discord channels
```

**Review by peterjgilbert — APPROVED** [review]

File: docs/general/foundation/delegation-policy.md (hunk 2 above)
Comment (tjanez) [reply docs/general/foundation/delegation-policy.md]: Updated

File: docs/general/foundation/delegation-policy.md (hunk 1 above)
Comment (tjanez) [reply docs/general/foundation/delegation-policy.md]: Updated

**Pull request merged**